packages/service-library/tests/deferred_tasks/test__redis_memory_manager.py

```diff
@@ -0,0 +1,81 @@
+# pylint:disable=redefined-outer-name
+# pylint:disable=unused-argument
+
+
+from collections.abc import AsyncIterable
+from datetime import timedelta
+
+import pytest
+from pydantic import parse_obj_as
+from servicelib.deferred_tasks._models import TaskUID
+from servicelib.deferred_tasks._redis_memory_manager import RedisMemoryManager
+from servicelib.deferred_tasks._task_schedule import TaskSchedule, TaskState
+from servicelib.redis import RedisClientSDKHealthChecked
+from servicelib.utils import logged_gather
+from settings_library.redis import RedisDatabase, RedisSettings
+
+pytest_simcore_core_services_selection = [
+    "redis",
+]
+
+
+@pytest.fixture
+async def scheduling_redis_sdk(
```

Comment:
instead of duplicating fixtures, can you please reuse them in conftest and also check if that does not already exist somewhere else? This is very painful for maintenance later on.

Also, you might think of using fakeredis.

Reply:
refactored to use common fixture